### PR TITLE
Take Item from Cable Plate Stack on Place

### PR DESCRIPTION
**Pull request by Thomas--S**
~_untested_~

### CODE REVIEW

File: technic/machines/register/cables.lua [hunk 1]
```diff
@@ -210,6 +210,8 @@ function technic.register_cable(tier, size)
 					end
 				end
 				minetest.set_node(pointed_thing.above, {name = "technic:"..ltier.."_cable_plate_"..num})
+				itemstack:take_item()
```
Comment (VanessaE) [inline technic/machines/register/cables.lua]: Need to check for creative mode (or priv) before doing that.

**Comment by Thomas--S** [comment]
I hope this works like expected. :wink:

**Comment by Thomas--S** [comment]
Tested now, should work like expected.

**Comment by Thomas--S** [comment]
Changed the drop to a recipe.

Tested again, everything works fine.

@VanessaE This is ready for merge in my opinion.

**Comment by Thomas--S** [comment]
Thanks for merging!

**Pull request merged**